eddymotion/estimator.py

```diff
@@ -132,6 +136,10 @@ def fit(
                         result = registration.run(cwd=str(tmpdir)).outputs
 
                         # read output transform
+                        aff_dict = loadmat(result.forward_transforms[0])
+                        aff_dict['AffineTransform_float_3_3'] = \
+                            aff_dict.pop('AffineTransform_double_3_3')
+                        savemat(result.forward_transforms[0], aff_dict)
```

Comment:
Is `.mat` the native format for these things? (produced by ANTS, I assume?)

Reply:
> This LGTM! Is the ANTS stuff a blocker to merging? Any chance you could add tests of the base classes and `get_run_cls`?

This is truly a band-aid solution, and the change should really be made in NiTransforms rather than here. But if we wait for a separate NiTransforms PR, it would probably delay the merge?

> Is `.mat` the native format for these things? (produced by ANTS, I assume?)
Yes (for ANTs transforms), but could easily be a .txt, .csv, or .npy instead.

Test for `get_run_cls` and base classes forthcoming.

Reply:
> This is truly a band-aid solution, and the change should really be made in NiTransforms rather than here. But if we wait for a separate NiTransforms PR, it would probably delay the merge?

Yup, this is polluting the PR, and it should work with the latest NiTransforms and recent ANTs (+2.3.3).